libs/auth-api-lib/src/lib/delegations/delegations-index.service.ts

```
@@ -30,17 +32,13 @@ import { DelegationDTO } from './dto/delegation.dto'
-import {
-  delegationProviderTypeMap,
-  validateDelegationTypeAndProvider,
-  validateToAndFromNationalId,
-} from './utils/delegations'
+import { DelegationRecordType, PersonalRepresentativeDelegationType } from './types/delegationRecord'
+import { validateDelegationTypeAndProvider, validateToAndFromNationalId } from './utils/delegations'
 import { getXBirthday } from './utils/getXBirthday'
 import { isUnderXAge } from './utils/isUnderXAge'
+import { ApiScopeDelegationType } from '../resources/models/api-scope-delegation-type.model'
+import { FeatureFlagService, Features } from '@island.is/nest/feature-flags'
+import { isEmpty } from 'lodash'
```

ESLint

[error] 'lodash' import is restricted from being used.

(no-restricted-imports)